src/Umbraco.Infrastructure/DependencyInjection/UmbracoBuilder.Services.cs

```diff
@@ -122,7 +122,11 @@ private static LocalizedTextServiceFileSources SourcesFactory(IServiceProvider c
             var pluginLangFolders = appPlugins.Exists == false
                 ? Enumerable.Empty<LocalizedTextServiceSupplementaryFileSource>()
                 : appPlugins.GetDirectories()
-                    .SelectMany(x => x.GetDirectories("Lang", SearchOption.AllDirectories))
+                    // The plugin lang folder was previously looked for as `Lang` with uppercase L but this causes inconsistencies
+                    // and problems on Linux, so we attempt something of a case-insensitive lookup instead. This is required in case
+                    // plugin developers have already released packages with the `Lang` folder, but we really want to suggest `lang` 
+                    // is better
```

Comment:
```suggestion
                    // Check for both Lang & lang to support case sensitive file systems. (Lang is preferred)
```

Reply:
The convention (when it actually gets documented / established / implemented) is that files which could be served by a static web server should all be lower/kebab case.

Lang files are picked up by C# code and made use of via an api call, so they are considdered backend files (which granted is a bit weird because they are found in App_Plugins which is essentially another frontend only folder).

Hopefully new back office will make all this less strange.

Reply:
👍🏻